src/geophires_x/Economics.py

```diff
@@ -112,6 +135,123 @@ def CalculateCostOfOneWell(model: Model, depth_m: float, well_correlation: int,
+        cost_per_section = (0.15340 * length_per_section_m ** 2 + 120.31700 * length_per_section_m + 1431801.54400) * 1E-6
+        cost_of_non_vertical_section = casing_factor * num_nonvertical_sections * cost_per_section
+    elif well_correlation is WellDrillingCostCorrelation.DEVIATED_LARGE_INT1:
+        cost_per_section = (0.19950 * length_per_section_m ** 2 + 296.13011 * length_per_section_m + 1697867.70900) * 1E-6
+        cost_of_non_vertical_section = casing_factor * num_nonvertical_sections * cost_per_section
+    elif well_correlation is WellDrillingCostCorrelation.VERTICAL_SMALL_INT2:
+        cost_per_section = (0.00804 * length_per_section_m ** 2 + 455.60507 * length_per_section_m + 921007.68680) * 1E-6
+        cost_of_non_vertical_section = casing_factor * num_nonvertical_sections * cost_per_section
+    elif well_correlation is WellDrillingCostCorrelation.VERTICAL_LARGE_INT2:
+        cost_per_section = (0.00315 * length_per_section_m ** 2 + 782.69676 * length_per_section_m + 983620.25270) * 1E-6
+        cost_of_non_vertical_section = casing_factor * num_nonvertical_sections * cost_per_section
+    elif well_correlation is WellDrillingCostCorrelation.DEVIATED_SMALL_INT2:
+        cost_per_section = (0.00854 * length_per_section_m ** 2 + 506.08357 * length_per_section_m + 1057330.39000) * 1E-6
+        cost_of_non_vertical_section = casing_factor * num_nonvertical_sections * cost_per_section
+    elif well_correlation is WellDrillingCostCorrelation.DEVIATED_LARGE_INT2:
+        cost_per_section = (0.00380 * length_per_section_m ** 2 + 838.90249 * length_per_section_m + 1181947.04400) * 1E-6
+        cost_of_non_vertical_section = casing_factor * num_nonvertical_sections * cost_per_section
+    elif well_correlation is WellDrillingCostCorrelation.VERTICAL_SMALL_IDEAL:
+        cost_per_section = (0.00252 * length_per_section_m ** 2 + 439.44503 * length_per_section_m + 590611.90110) * 1E-6
+        cost_of_non_vertical_section = casing_factor * num_nonvertical_sections * cost_per_section
+    elif well_correlation is WellDrillingCostCorrelation.VERTICAL_LARGE_IDEAL:
+        cost_per_section = (-0.00240 * length_per_section_m ** 2 + 752.93946 * length_per_section_m + 524337.65380) * 1E-6
+        cost_of_non_vertical_section = casing_factor * num_nonvertical_sections * cost_per_section
+    elif well_correlation is WellDrillingCostCorrelation.DEVIATED_SMALL_IDEAL:
+        cost_per_section = (0.00719 * length_per_section_m ** 2 + 455.85233 * length_per_section_m + 753377.73080) * 1E-6
+        cost_of_non_vertical_section = casing_factor * num_nonvertical_sections * cost_per_section
+    elif well_correlation is WellDrillingCostCorrelation.DEVIATED_LARGE_IDEAL:
+        cost_per_section = (0.00376 * length_per_section_m ** 2 + 762.52696 * length_per_section_m + 765103.07690) * 1E-6
+        cost_of_non_vertical_section = casing_factor * num_nonvertical_sections * cost_per_section
```

Comment:
Are these correlations copied from existing code or do the values differ?

Reply:
Yes, this is a cut and paste from another section of code.